src/brevitas/quant_tensor/__init__.py

```diff
@@ -298,7 +298,7 @@ def cat(tensors, dim, out=None):
             if all([isinstance(qt, QuantTensor) and qt.is_not_none for qt in tensors]):
                 for qt in tensors[1:]:
                     first_qt.check_scaling_factors_same(qt)
-                    first_qt.check_zero_points_same(qt)
+                    first_qt.check_zero_points_same(qt) # Torch.allclose fails with different data types
```

Comment:
Not clear to me what this comment is trying to say

Reply:
if you have two zero point tensors with different data types (e.g., torch.uint8 and torch.int, even if both tensors are 0), this check fails

Reply:
I would clarify then the check can failr when the zero point data types are different. Torch.allclose doesn't show up here